packages/app/components/feed/SearchProvider.tsx

```diff
@@ -0,0 +1,16 @@
+import React, { createContext, useState } from 'react';
+
+// Create a context with default value
+const SearchContext = createContext();
```

Comment:
The search is working  on native but the useContext is undefined on web. I think createContext should be provided some default value to avoid this.

Reply:
Wrap the /screen component in context not the expo router should fix this